src/org/labkey/test/tests/SampleSetRemoteAPITest.java

```diff
@@ -132,6 +143,253 @@ public void samplesWithLookupsToExpFilesTest() throws IOException, CommandExcept
+
+        dgen.deleteDomain(createDefaultConnection(true));
+    }
+
+    @Test
+    public void importMVDataWithEmptyValues() throws IOException, CommandException
+    {
+        String missingValueTable = "mvSamplesWithEmptyValues";
+
+        navigateToFolder(getProjectName(), FOLDER_NAME);
+        TestDataGenerator dgen = new TestDataGenerator("exp.materials", missingValueTable, getCurrentContainerPath())
+                .withColumnSet(List.of(
+                        TestDataGenerator.simpleFieldDef("name", FieldDefinition.ColumnType.String),
+                        TestDataGenerator.simpleFieldDef("mvStringData", FieldDefinition.ColumnType.String)
+                                .setMvEnabled(true).setLabel("MV Field"),
+                        TestDataGenerator.simpleFieldDef("volume", FieldDefinition.ColumnType.Double)
+                ))
+                .withGeneratedRows(30);
+        dgen.createDomain(createDefaultConnection(true), "SampleSet");
+        dgen.addCustomRow(Map.of("name", "First", "mvStringData", "Q", "volume", 13.5));
+        dgen.addCustomRow(Map.of("name", "Second", "mvStringData", "Q", "volume", 15.5));
+        dgen.addCustomRow(Map.of("name", "Third","mvStringData", "N", "volume", 16.5));
+        dgen.addCustomRow(Map.of("name", "Fourth","mvStringData", "N", "volume", 17.5));
+        dgen.addCustomRow(Map.of("name", "Fifth","mvStringData", "", "volume", 17.5));
+        dgen.addCustomRow(Map.of("name", "Sixth","mvStringData", "", "volume", 17.5));
+        dgen.addCustomRow(Map.of("name", "Seventh","mvStringData", "ValidValue", "volume", 17.5));
+        dgen.addCustomRow(Map.of("name", "Eighth","mvStringData", "ActualData", "volume", 17.5));
+
+        // insert them via API
+        dgen.insertRows(createDefaultConnection(true), dgen.getRows());
+
+        // now find the samlpleset's dataregion
+        refresh();
+        DataRegionTable sampleSetList =  DataRegionTable.DataRegion(getDriver()).withName("SampleSet").waitFor();
+        waitAndClick(Locator.linkWithText(missingValueTable));
+        DataRegionTable materialsList =  DataRegionTable.DataRegion(getDriver()).withName("Material").waitFor();
+
+        // confirm that fifthRow has no MV indicator supplied
+        int mvColumnIndex = materialsList.getColumnIndex("MV Field");
+        int fifthRowIndex = materialsList.getRowIndex("Name", "Fifth");
+        WebElement fifthRowMvCell = materialsList.findCell(mvColumnIndex, fifthRowIndex);
+        assertEquals("", fifthRowMvCell.getAttribute("class"));
+        // ensure that fourthRow has an MV indicator with value N
+        int fourthIndex = materialsList.getRowIndex("Name", "Fourth");
+        WebElement fourthRow = materialsList.findRow(fourthIndex);
+        Locator.tagWithClass("td", "labkey-mv-indicator")
+                .withDescendant(Locator.tagWithText("a", "N")).findElement(fourthRow);
+
+        dgen.deleteDomain(createDefaultConnection(true));
+    }
+
+    @Test
+    @Ignore
```

Comment:
counteroffer; I've removed the @Ignore, this will fail until import/export supports missing values.
There are several issues (that have been fixed, closed) on this, but it appears to have regressed

Reply:
Merging a broken test is more painful to the test team than anybody else. If there isn't somebody actively working to fix the issue, we shouldn't just leave this test failing indefinitely. Open/re-open an issue for this and re-add the `@ignore` with a link to the issue

Reply:
Okay, new bug is written, issue is re-ignored with a note about which bug it is